lively.halos/morph.js

```diff
@@ -123,19 +123,21 @@ export default class Halo extends Morph {
   initButtons () {
     this.submorphs = [
       ...this.ensureResizeHandles(),
-      ...this.resizeOnly ? [] : [
-        this.closeHalo(),
-        this.dragHalo(),
-        this.grabHalo(),
-        this.menuHalo(),
-        this.inspectHalo(),
-        this.editHalo(),
-        this.copyHalo(),
-        this.componentHalo(),
-        this.rotateHalo(),
-        this.nameHalo(),
-        this.originHalo()
-      ]
+      ...this.resizeOnly
+        ? []
+        : [
```

Comment:
This was done everywhere by the linter, I assume?

Reply:
correct